SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Policy
+
+## Supported Versions
+
+The OpenIndiana project commits to providing fixes or mitigations to each
+and every security vulnerabilities in software we deliver. Such a repositories
+of software sources are [illumos-gate](https://github.com/illumos/illumos-gate),
+[pkg5](https://github.com/OpenIndiana/pkg5), [slim_source](https://github.com/OpenIndiana/slim_source),
```

Comment:
Perhaps, better include link on https://github.com/OpenIndiana/ directly or on forthcoming OpenIndiana software repositories link on oi-docs site.  Also we should mentioned that we deliver independently developed software, and issues in such software also should be reported.